sci-electronics/kicad/kicad-4.0.0_rc1.ebuild

```diff
@@ -0,0 +1,183 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+PYTHON_COMPAT=( python2_7 )
+
+WX_GTK_VER="3.0"
+
+inherit cmake-utils fdo-mime flag-o-matic gnome2-utils python-single-r1 python-utils-r1 vcs-snapshot wxwidgets versionator
+
+DESCRIPTION="Electronic Schematic and PCB design tools."
+HOMEPAGE="http://www.kicad-pcb.org"
+
+SERIES=$(get_version_component_range 1-2)
+MY_PV=$(replace_version_separator 3 '-')
+
+LIB_REVISION="bd0ec6a20c7cc0556093f60eac723e4505947784"
+DOC_REVISION="0d0ed7b9de3afbba5d51fb6d7a25834000ed1099"
+I18N_REVISION="bf793d30d64f4e75ea7a109ea434c1a27926b158"
+
+SRC_URI="https://launchpad.net/${PN}/${SERIES}/${MY_PV}/+download/${PN}-${MY_PV}.tar.xz
+	!minimal? ( https://github.com/KiCad/${PN}-library/tarball/${LIB_REVISION} -> ${P}-library.tar.gz )
+	i18n? ( https://github.com/KiCad/${PN}-i18n/tarball/${I18N_REVISION} -> ${P}-i18n.tar.gz )"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc examples github i18n minimal python webkit"
+LANGS="bg ca cs de el es fi fr hu it ja ko nl pl pt ru sk sl sv zh_CN"
+for lang in ${LANGS} ; do
+	IUSE="${IUSE} linguas_${lang}"
+done
+unset lang
+
+REQUIRED_USE="
+	github? ( webkit )
+	python? ( ${PYTHON_REQUIRED_USE} )"
+
+CDEPEND="x11-libs/wxGTK:${WX_GTK_VER}[X,opengl,webkit?]
+	python? (
+		dev-python/wxpython:${WX_GTK_VER}[opengl,${PYTHON_USEDEP}]
+		${PYTHON_DEPS}
+	)
+	>=dev-libs/boost-1.56[context,threads,python?]
+	github? ( dev-libs/openssl:0 )
+	media-libs/glew
+	media-libs/freeglut
+	media-libs/mesa
+	sys-libs/zlib
+	x11-libs/cairo"
+DEPEND="${CDEPEND}
+	app-arch/xz-utils
+	doc? ( app-doc/doxygen )
+	i18n? ( >=sys-devel/gettext-0.18 )
+	python? ( dev-lang/swig:0 )"
+RDEPEND="${CDEPEND}
+	sci-electronics/electronics-menu"
+
+S="${WORKDIR}/${PN}-${MY_PV}"
+
+pkg_setup() {
+	python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	if use python; then
+		# dev-python/wxpython doesn't support python3
+		sed '/set(_PYTHON3_VERSIONS 3.3 3.2 3.1 3.0)/d' -i CMakeModules/FindPythonLibs.cmake || die "sed failed"
```

Comment:
I'd cut this on version list, to avoid having it silently break when new versions are added.